driver/pace/driver/run.py

```diff
@@ -1,12 +1,13 @@
 import os
 import sys
 from dataclasses import fields
-from typing import List, Literal, Optional, Tuple
+from typing import List, Optional, Tuple
 
 import click
 import f90nml
 import numpy as np
 from mpi4py import MPI
+from typing_extensions import Literal
```

Comment:
nit: It's only necessary to import this from typing_extensions if we're on python below 3.8, but since this depends on gt4py and gt4py depends on python >=3.8 we should be able to import it from `typing`. I don't mind either way but wanted to make sure you know this isn't necessary for the versions we use.

Reply:
ah it was complaining at me because I'm running the linter using python 3.7 then. we import it from typing_extensions in another part of the code...

Reply:
if you want to leave it to get it in for now that’s fine, there isn’t really any harm in it.